packages/squarev2-integration/src/SquareV2PaymentMethod.tsx

```diff
@@ -0,0 +1,199 @@
+import { difference } from 'lodash';
+import React, { FunctionComponent, useCallback, useEffect, useState } from 'react';
+
+import {
+    PaymentMethodProps,
+    PaymentMethodResolveId,
+    toResolvableComponent,
+} from '@bigcommerce/checkout/payment-integration-api';
+import { getAppliedStyles, LoadingOverlay } from '@bigcommerce/checkout/ui';
+
+const SquareV2PaymentMethod: FunctionComponent<PaymentMethodProps> = ({
+    method,
+    checkoutService,
+    checkoutState,
+    paymentForm,
+}) => {
+    const renderDummyFormFields = () => {
```

Comment:
Can you please explain the use of `dummy` form fields?

Reply:
Sorry think I confused you :|, I was wondering if we can do better naming as compared to `dummy`. Are these just placeholder fields?

Reply:
Can we name them `placeholderFields`?

Reply:
Yes, that sounds better. Thanks